src/dap_on_summary_stats.py

```diff
@@ -51,8 +51,7 @@ def _dap_command(region, intermediate_folder, output_folder, options, dap_comman
 {dap} \\
 -d_z {stats} \\
 -d_ld {cor} \\
-{extra} \\
```

Comment:
making the `-t 1` an option will "break" old scripts from gtex v8 analysis, if someone tries to run them again they will use multithreading and choke the cluster to death. Please put a note somewhere on the gtexv8 readme about this change.